pytest_reportportal/plugin.py

```diff
@@ -39,7 +46,14 @@ def pytest_configure(config):
             "pytest report portal is not compatible with 'xdist' plugin.")
 
     # set Pytest_Reporter and configure it
-    config._reporter = RPReportListener()
+
+    if PYTEST_HAS_LOGGING_PLUGIN:
+        # This check can go away once we support pytest >= 3.3
+        config._reporter = RPReportListener(
+            _pytest.logging.get_actual_log_level(config, 'rp_log_level')
```

Comment:
When log level is not specified, `_pytest.logging.get_actual_log_level(config, 'rp_log_level')` returns `None` and as result:
```
INTERNALERROR> TypeError: Level not an integer or a valid string: None
```